CHANGELOG.md

```diff
@@ -25,6 +25,9 @@ https://github.com/bitcrowd/sshkit.ex/compare/v0.2.0...HEAD
 ### Fixes:
 
 <!-- Put fixes here -->
+* Add support for initializing subsystems using `SSH.Channel.subsystem`
+  * Subsystems allow applications or functions to use SSH as a protocol.
+    Examples of this are SFTP and NETCONF over SSH.
```

Comment:
I love the explanation why one might want to support subsystems ❤️ 
I'd have categorized this as a new feature rather than a fix 😁🤷‍♂